apps/contract-sync-job/src/main.ts

```diff
@@ -115,34 +126,34 @@ const main = async () => {
       async contribution => await upsertContribution(contribution),
       BATCH_SIZE,
     );
-    console.error('batch Contribution errors', errors);
+    logger.error(':: BATCH CONTRIBUTION ERRORS', errors);
 
     const upsertedCount = inserted.length;
     if (upsertedCount > 0) {
-      console.log(`:: Inserted ${upsertedCount} Contribution(s)`);
+      logger.info(`:: Inserted ${upsertedCount} Contribution(s)`);
```

Comment:
Do you think that `::` suffix is still necessary?

Reply:
no, probably not, i can remove it in an additional PR if you like